src/program.js

```diff
@@ -176,6 +195,8 @@ type VersionGetterOptions = {
   globalEnv?: string,
 };
 
+export type VersionGetterFn = (absolutePackageDir: string) => string;
```

Comment:
@saintsebastian nit: do you mind to move this flow type definition before the first usage at line 33?